### ISSUE TITLE
Please add site https://https://novtales.com/

### ISSUE THREAD

**Issue by lomborton** [dteviot/WebToEpub#1296]
Please note, I'm basically the only developer working on WebToEpub, and I'm not paid for doing this.  (WebToEpub is completely free, and generates no money.) By asking to add a site, you're asking me to give you some of my limited free time. So, I think it's not unreasonable for me to ask you to do as much as you can to help me.

### Did you try using the Default Parser for the site?  If not, why not?
Instructions for using the default parser can be found at https://dteviot.github.io/Projects/webToEpub_DefaultParser.html

### What settings did you use?  What didn't work?
* URL of first chapter
* CSS selector for element holding content to put into EPUB
* CSS selector for element holding Title of Chapter
* CSS selector for element(s) to remove

### If the Default Parser did not work, if you have developer skills, did you try writing a new parser?
Instructions https://dteviot.github.io/Projects/webToEpub_FAQ.html#write-parser

**Comment by lomborton** [comment]
https://novtales.com/novel/cultivating-immortality-in-the-chaotic-world-of-demons-and-martial-arts

**Comment by lomborton** [comment]
I tried everything, but it didn't work.

**Comment by dteviot** [comment]
@lomborton 
> I tried everything, but it didn't work.

I'm not surprised.  That site is difficult.
1. The table of contents does not contain links to chapters.  It's just &lt;div&gt; elements. Chapter URLs are probably derived by some javascript function. Note, I suspect it's **mostly** a simple case of, if the ToC page is
https://novtales.com/novel/cultivating-immortality-in-the-chaotic-world-of-demons-and-martial-arts,
then the chapters are
https://novtales.com/chapter/cultivating-immortality-in-the-chaotic-world-of-demons-and-martial-arts-1

i.e. replace /novel/ with /chapter/ and then add suffix for chapter at end.  But I don't know if all chapters follow that pattern.

2. Chapters theselves are a much bigger problem,  The actual chapter text is obtained by POSTing a REST call to https://novtales.com/elasticsearch/msearch, but there are a lot of parameters in the call, and again, it's not obvious what values to put in.  Probably want to brute force it, just open each page in a browser, let it do the hard work and scrape the text out when done.  Which WebToEpub does not do.

TL;DR;
Site too complicated for me to be willing to get WebToEpub to handle it.

For my notes: 18 minutes work
